samples/attested_tls/server/enc/openssl_server.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <arpa/inet.h>
+#include <openenclave/enclave.h>
+#include <openssl/evp.h>
+#include <openssl/ssl.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include "../../common/openssl_utility.h"
+
+extern "C"
+{
+    int setup_tls_server(char* server_port, bool keep_server_up);
+};
+
+int verify_callback(int preverify_ok, X509_STORE_CTX* ctx);
+
+int initalize_ssl_context(SSL_CTX*& ctx)
+{
+    int ret = -1;
+    const SSL_METHOD* method;
+    if ((ctx = SSL_CTX_new(SSLv23_server_method())) == NULL)
+    {
+        printf(TLS_SERVER " unable to create a new SSL context\n");
+        goto exit;
+    }
+    // choose TLSv1.2 by excluding SSLv2, SSLv3 ,TLS 1.0 and TLS 1.1
```

Comment:
Is TLSv1.3 supported or not?  https://www.zdnet.com/article/openssl-1-1-1-out-with-tls-1-3-and-complete-rewrite-of-rng-component says openssl 1.1.1 does support TLS 1.3
```suggestion
    // choose TLSv1.2 by excluding SSLv2, SSLv3, TLS 1.0, and TLS 1.1
```

Reply:
Used TLS_server_method/TLS_client_method which negotiates to the highest version possible on the server/client. Exisitng Openssl code show max version used is TLS 1.3 version. https://github.com/openssl/openssl/blob/54a0d4ceb28d53f5b00a27fc5ca8ff8f0ddf9036/ssl/methods.c#L23